stub_uploader/metadata.py

```diff
@@ -231,6 +231,7 @@ def verify_typeshed_req(req: Requirement) -> None:
     "torch",
     "tree-sitter",
     "urllib3",
+    "pyproj",
```

Comment:
Please note that the list is sorted alphabetically (except that capitalized names are first).